xbmc.py

```diff
@@ -58,6 +57,7 @@ def __init__(self):
         # noinspection PyUnusedLocal
         def stop_requested(signum, frame):
             self.__abort = True
+            Player.stop()
```

Comment:
The `Player` has nothing to do with the monitor?

Reply:
When the Player is Playing, and you request a shutdown. It should stop.

Reply:
True, but that is something Kodi handles, and is not done via the Python code.

Reply:
Hmm, but isn't this Python code emulating what Kodi does?

I actually added this since when I ran tests that didn't stop playback from inside the test, it kept running, and since the Fake Player runs on a different thread, the test kept running untill the thread was terminated. It would seem like a clean idea to Stop the player on shutdown, like the real Kodi probably also does.